packages/exo/src/exo-makers.js

```diff
@@ -222,7 +238,6 @@ harden(defineExoClassKit);
  * }> | undefined} interfaceGuard CAVEAT: static typing does not yet support `callWhen` transformation
  * @param {T} methods
  * @param {FarClassOptions<ClassContext<{},T>>} [options]
- * @returns {T & import('@endo/eventual-send').RemotableBrand<{}, T>}
```

Comment:
while this may not be necessary for the type checker or IDE, it's helpful for reading the code. Is this inaccurate now or just deemed superfluous?

Reply:
Okay.  It was both inaccurate and deemed superfluous.  I've added correct `@returns` types instead.